snmp/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+***Fixed***:
+
+* Fix SNMP profiles metric tags mappings using MIBs ([#15668](https://github.com/DataDog/integrations-core/pull/15668))
```

Comment:
```suggestion
* Fix SNMP profiles metric tags mappings using MIBs ([#15668](https://github.com/DataDog/integrations-core/pull/15668))

```

Reply:
addressed in [eef9c48](https://github.com/DataDog/integrations-core/pull/15668/commits/eef9c48a692f77111382b7b819446a12d69a5c25)